Submissions/Ekey_Week2 Simple Values and Control Flow.playground/Contents.swift

```diff
@@ -0,0 +1,72 @@
+//: Simple Values
+//: 1. Create a constant with an explicit type of Float that equals 4.
+let explicitFloat: Float = 4.0
+
+//: 2. Try removing the conversion to String from the last line. Copy the error and place it in a block comment below this block of code.
+let label = "The width is "
+let width = 94
+let widthLabel = label + String(width)
+
+//  let widthLabel = label + (width)
+//  Binary operator '+' cannot be applied to operands of type 'String' and 'Int'
+
+//: 3. Use \() to include a floating-point calculation in a string and to include someone's name in a greeting.
+let myName = "Laurie"
+let Greeting = "Hello my name is "
+let age: Float = 52.1
+let assignment1 = "\(Greeting)\(myName) and I am OLD, \(age)"
+print(assignment1)
+
+//: Control Flow
+//: 1. Change optional Name to nil. What greeting do you get? Add an else clause that sets a different greeting if optionalName is nil.
```

Comment:
The question was what greeting do you get, not what does optionalName print. If `optionalName` is nil and there is no else clause, then the declaration of `greeting` is "Hello!" and it is never modified as the `if let` block is never executed, so the greeting we get is "Hello!"